libs/s25main/controls/ctrlImageButton.cpp

```diff
@@ -11,11 +11,18 @@ ctrlImageButton::ctrlImageButton(Window* parent, unsigned id, const DrawPoint& p
 
 void ctrlImageButton::DrawContent() const
 {
-    DrawPoint pos = GetDrawPos() + DrawPoint(GetSize()) / 2;
+    DrawPoint pos = GetDrawPos() - GetImageRect().getOrigin() + DrawPoint::all(2);
```

Comment:
Please add a comment here how/where the image is supposed to be drawn, i.e. why the offset of 2 and 4 at various places. Maybe introduce some named constants for those values.